<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDlg</class>
 <widget class="QDialog" name="SettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>270</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QPushButton" name="okPB">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="colorGrpBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Color</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>Color</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>84</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="blueRadioBtn">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="redRadioBtn">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="levelGrpBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Level</string>
   </property>
   <widget class="QRadioButton" name="beginnerRadioBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Beginner</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="profiRadioBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Professional</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="masterRadioBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Master</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelPB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Changes will be applied only after starting a new game</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
